openedx/core/djangoapps/course_groups/tests/test_partition_scheme.py

```diff
@@ -359,13 +359,15 @@ def _verify_masquerade_for_group(self, group):
         }
         request = self._create_mock_json_request(
             self.test_user,
-            body=json.dumps(request_json),
+            data=request_json,
             session=self.session
         )
-        handle_ajax(request, unicode(self.course.id))
+        response = handle_ajax(request, unicode(self.course.id))
+        # pylint has issues analyzing this class (maybe due to circular imports?)
+        self.assertEquals(response.status_code, 200)  # pylint: disable=no-member
 
         # Now setup the masquerade for the test user
-        setup_masquerade(request, self.test_user, True)
+        setup_masquerade(request, self.course.id, True)
```

Comment:
Was this just an error before?  I don't see anywhere that the signature changes for setup_masquerade...